v2/tools/generator/internal/astmodel/compat_package_reference.go

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astmodel
+
+func MakeCompatPackageReference(pkg InternalPackageReference) SubPackageReference {
```

Comment:
Function documentation here would be helpful here for future ref - on why and where is this used